pkg/gofr/gofr.go

```diff
@@ -33,7 +33,10 @@ import (
 
 const (
 	defaultPublicStaticDir = "static"
-	gofrTraceExporter      = "gofr"
+	traceExporterGoFr      = "gofr"
+	traceExporterZipkin    = "zipkin"
+	traceExporterJaeger    = "jaeger"
+	traceExporterOTLP      = "otlp"
```

Comment:
@Sianao - if we are adding support for a new exporter name `otlp`, this should also be updated in docs

Reply:
All done